library/akamai.py

```diff
@@ -1,29 +1,130 @@
-#!/usr/bin/env python
+#!/usr/bin/python
```

Comment:
I believe keeping `/usr/bin/env python` will perform correctly but has an added advantage in some environments.

https://www.baeldung.com/linux/bash-shebang-lines.

Is there a specific reason to change it?

Reply:
Not a major preference, just following [the ansible guidelines](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#python-shebang-utf-8-coding).

Reply:
Oddly enough, the ansible-core native modules don't include shebang at all. [Example](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/copy.py)